tests/cupyx_tests/linalg_tests/sparse_tests/test_solve.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+
+import numpy
+try:
+    import scipy.linalg
+    import scipy.sparse
+    import scipy.stats
+
+    scipy_available = True
+except ImportError:
+    scipy_available = False
+
+import cupy as cp
+from cupy import cuda
+import cupy.sparse as sp
+from cupy import testing
+from cupy.testing import condition
+import cupyx
+
+
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float32, numpy.float64],
+}))
+@unittest.skipUnless(
+    cuda.cusolver_enabled, 'Only cusolver in CUDA 8.0 is supported')
+@unittest.skipUnless(scipy_available, 'requires scipy')
+class TestLschol(unittest.TestCase):
+
+    def setUp(self):
+        rvs = scipy.stats.randint(0, 15).rvs
+        self.A = scipy.sparse.random(
+            50, 50, density=0.2, data_rvs=rvs, dtype=self.dtype)
+        self.b = numpy.random.randint(15, size=50)
+        self.A = self.A.T*self.A  # symmetric and positive definite
+        self.b = self.A.T*self.b
+        # inital scipy results by dense cholesky method.
+        L = scipy.linalg.cho_factor(self.A.todense())
+        self.x = scipy.linalg.cho_solve(L, self.b)
+        if self.dtype == numpy.float64:
+            self.decimal = 6
+        else:
+            self.decimal = 3
```

Comment:
Sometimes each test case cannot pass with this condition. Please reduce `decimal` or fix random values.

Reply:
By strengthing the diagnal elements of the test matrix, the positive definiteness is ensured and the condition number decreased a lot. Therefore, the test results should be stable now.